gradle-module/src/main/java/org/example/Main.java

```diff
@@ -5,21 +5,23 @@ public class Main {
     public static final String CONSTANT = "constant";
```

Comment:
[Severity: Low] The class does not have a Javadoc comment explaining its purpose.
 Consider adding a Javadoc comment at the class level to describe its functionality and usage.
 Rationale: Javadoc comments provide essential information for developers using or maintaining the class, enhancing overall code documentation.